src/sagemaker/serve/builder/transformers_builder.py

```diff
@@ -99,7 +100,27 @@ def _create_transformers_model(self) -> Type[Model]:
         if hf_model_md is None:
             raise ValueError("Could not fetch HF metadata")
 
-        if "pytorch" in hf_model_md.get("tags"):
+        model_task = None
+        if self.model_metadata:
+            model_task = self.model_metadata.get("HF_TASK")
```

Comment:
what if the metadata doesn't have `HF_TASK` field? I think we should follback to the pipeline_tag path

Reply:
yeah let me update